packages/client/hmi-client/src/composables/useCharts.ts

```diff
@@ -1230,3 +1250,38 @@ export function useCharts(
 		useEnsembleErrorCharts
 	};
 }
+
+export function setInterventionColorAndScoreMaps(
+	datasets,
+	modelConfigurations,
+	interventionPolicies,
+	interventionNameColorMap,
+	interventionNameScoresMap
+) {
+	let colorIndex = 0;
+	datasets.value.forEach((dataset) => {
+		const { metadata } = dataset;
+		const modelConfiguration: ModelConfiguration = modelConfigurations.value.find(
+			({ id }) => id === metadata.simulationAttributes?.modelConfigurationId
+		);
+		const policy: InterventionPolicy = interventionPolicies.value.find(
+			({ id }) => id === metadata.simulationAttributes?.interventionPolicyId
+		);
+
+		const policyName = policy?.name ?? 'no policy';
+
+		if (!modelConfiguration?.name) {
+			return;
+		}
+
+		if (!interventionNameColorMap[policyName]) {
+			interventionNameScoresMap[policyName] = [];
```

Comment:
The value of  `interventionNameScoresMap` isn't used inside this function. Why do we need this inside of this function? Can we make this function only work with the color mapping if it doesn't do anything with the scores?